go.sum

```diff
@@ -1,30 +1,62 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
There are a lot of changes here, and they are all additions -- no deletions. That is a little bit unsurprising.

Can you try locally deleting this file and then running `go test ./...`, to re-generate from scratch (and possibly prune some older no-longer-needed entries)?

Reply:
Nice catch. This caught some duplicate (unnecessary) dependencies in the file. Removed.